packages/@tinacms/api-git/src/utils/paths.test.ts

```diff
@@ -0,0 +1,35 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as path from 'path'
+
+import { checkFilePathIsInParent } from './paths'
+
+describe('checkFilePathIsInParent', () => {
```

Comment:
We run the tests on windows too. We could probably have a couple tests to check the behaviour on windows.

Reply:
I'm not sure what you mean. These tests do run on windows and verify that the behaviour is the same as on linux.